### PR TITLE
Prototype of rewritten Administration > Users and Administration > Rooms

### PR DESCRIPTION
**Pull request by gabriellsh**

### CODE REVIEW

File: client/components/basic/VerticalBar.css [hunk 1]
```diff
@@ -0,0 +1,13 @@
+.rcx-vertical-bar {
+	&--mobile {
+		position: absolute;
+		width: 100% !important;
+		height: 100%;
+	}
+
+	&--small {
+		position: absolute;
+		height: 100%;
```
Comment (sampaiodiego) [inline client/components/basic/VerticalBar.css:10]: Codacy Issue found: [Expected an empty line before property "height" (order/properties-order)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5350567)

File: client/components/basic/VerticalBar.css [hunk 2]
```diff
@@ -0,0 +1,13 @@
+.rcx-vertical-bar {
+	&--mobile {
+		position: absolute;
+		width: 100% !important;
```
Comment (sampaiodiego) [inline client/components/basic/VerticalBar.css:4]: Codacy Issue found: [Expected an empty line before property "width" (order/properties-order)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5350567)

File: app/ui/client/views/app/components/Directory/Table.css [hunk 3]
```diff
@@ -0,0 +1,10 @@
+.rcx-table__cell--header {
+	position: sticky;
+	top: 0;
+	background: white;
+}
+
+.rcx-avatar__element {
+	position: relative;
+	/* z-index: -1; */
```
Comment (sampaiodiego) [inline app/ui/client/views/app/components/Directory/Table.css:9]: Codacy Issue found: [Expected empty line before comment (comment-empty-line-before)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5350567)

File: app/ui/client/views/app/components/Directory/Table.css [hunk 4]
```diff
@@ -0,0 +1,10 @@
+.rcx-table__cell--header {
+	position: sticky;
+	top: 0;
+	background: white;
```
Comment (sampaiodiego) [inline app/ui/client/views/app/components/Directory/Table.css:4]: Codacy Issue found: [Expected an empty line before property "background" (order/properties-order)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5350567)

File: client/components/basic/VerticalBar.css [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+		height: 100%;
+	}
+
+	&--small {
+		position: absolute;
+		height: 100%;
+		right: 0;
```
Comment (sampaiodiego) [inline client/components/basic/VerticalBar.css:11]: Codacy Issue found: [Expected an empty line before property "right" (order/properties-order)](https://app.codacy.com/manual/RocketChat/Rocket-Chat/pullRequest?prid=5350567)

File: client/components/basic/VerticalBar.css (hunk 1 above)
Comment (ggazzo) [reply client/components/basic/VerticalBar.css:10]: SAI DAQUI DIEGO

**Pull request merged**